### ISSUE TITLE
v9 Ignoring config.js

### ISSUE THREAD

**Issue by QuantumLeaper** [moneroexamples/openmonero#108]
Was just playing around with v9, and noticed the frontend does not seem to listen to js/config.js

Is this due to the WASM file from https://github.com/mymonero/mymonero-core-js?

**Comment by moneroexamples** [comment]
yes. Not all what is in js/config.js being used now. With time, probably will need to clean this file, and throw out what is not-used.

**Comment by QuantumLeaper** [comment]
Alright, cool - was just making sure I wasn't crazy. Do you know of the location to change apiUrl: "http://127.0.0.1:1984/" in the WASM? I'm getting "Error connecting to the backend. Can't login."

**Comment by moneroexamples** [comment]
This acctualy is used in js/config.js :-) So you should not be able to do anything, unless you want to use hook-up mymonero-app-js mobile wallet to openmonero's backend.

just make sure network type is set as well. if backend running on stagnet, stagenet should also be used in js/config.js.

**Comment by QuantumLeaper** [comment]
Ahh, OK - got it .. kind-of working, haha.

Hey, was just playing around with some slight modifications to mymonero-core-js. After compiling the wasm, I get the following error when trying to use the newly compiled wasm file:
``mymonero-core.js?1:8 wasm streaming compile failed: LinkError: WebAssembly Instantiation:  table import 85 has a larger maximum size 45e than the module's declared maximum 1110``

Does mymonero-core.js need to be edited? The code looks like it's some type of emscriptten format, is there a way to regenerate this file?

**Comment by moneroexamples** [comment]
Did you use `bin/package_browser_js` to generate wasm and corresponding mymonero-core.js file? You can also ask at https://github.com/mymonero/mymonero-core-js as probably they will now more about wasm things.

**Comment by QuantumLeaper** [comment]
Actually, one more question relevant to this topic.
How did you generate the emscriptten file for html/js/crypto.js? Did you just compile crypto/crypto.cpp?

**Comment by moneroexamples** [comment]
I did not. I downloaded it from https://mymonero.com/js/crypto.js?1 

Some info about its origin are here: https://github.com/mymonero/mymonero-core-js/issues/7

New emscriptten code from mymonero eliminates lots of things, and I need to find time to go through all the legacy stuff to see what is actually being used these days and what has become obsolete. I'm not even sure that crypto.js is used this days?
